README.rst

```diff
@@ -157,6 +157,8 @@ along with their affiliation:
 
 .. _`University of Calgary`: http://www.ucalgary.ca/
 
+.. _`Tom Baker Cancer Centre`: https://www.ahs.ca/tbcc
```

Comment:
The documentation testing produced the following error:

![image](https://user-images.githubusercontent.com/6559099/59145174-97fdec80-8a23-11e9-949c-cb43a99cf829.png)

Meaning, you had referenced "Tom Baker Cancer Centre" within your name, but didn't actually provide a URL for it. Is this URL okay?

Reply:
Hi Simon, 
Sorry for the late reply. It's that time of the month when I have to do some monthly QA on the linacs and finish really late. That URL works great but if it is a problem we can remove it and just keep the u of c one. I have been trying to run yarn as superuser on my linux box but that didn't do the trick. I tried "yarn run tree" but it also gave me the same result as "yarn tree".

Reply:
No need to be sorry. :). The URL isn't a problem at all. It was just erroring out because it wasn't included. You can include both logos if you would like also. I think we'll need to squish down the space the logos are taking up soon by shrinking them down and putting multiple on the same line. But that's a good thing.